apps/prairielearn/src/middlewares/authzCourseOrInstance.js

```diff
@@ -164,14 +165,15 @@ module.exports = asyncHandler(async (req, res, next) => {
       res.clearCookie(override.cookie);
     });
 
-    let err = error.make(403, 'Access denied');
-    err.info = html`
-      <p>
-        You must be a member of the course staff in order to change the effective user. All
-        requested changes to the effective user have been removed.
-      </p>
-    `.toString();
-    throw err;
+    throw new error.AugmentedError('Access denied', {
```

Comment:
Not that I mind, but I think we'd normally just import `AugmentedError` directly?

Reply:
Yep, this was me + Copilot being lazy 😂 I'll update this to do the right thing here.

Reply:
Done in ea136d0.